MapboxNavigation/NavigationViewController.swift

```diff
@@ -354,6 +354,7 @@ public class NavigationViewController: UIViewController, RouteMapViewControllerD
         mapViewController.reportButton.isHidden = !showsReportFeedback
 
         self.currentStyleType = styleTypeForTimeOfDay
+        self.voiceController?.routeController = self.routeController
```

Comment:
I'm worried about the voice controller being inited before the routeController is set on it. Should this line 
https://github.com/mapbox/mapbox-navigation-ios/blob/6dce036818e23d00d47efd7a8897fc703da61b97/MapboxNavigation/NavigationViewController.swift#L217

Be moved inside of the init method here?